docs/utils/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+load("//tools:defaults.bzl", "ts_library")
+
+ts_library(
+    name = "utils",
+    srcs = [
+        "index.ts",
+    ],
+    visibility = ["//docs:__subpackages__"],
+    deps = [
+        ":lib",
+    ],
+)
+
+ts_library(
+    name = "testing-helper",
+    srcs = [
+        "testing-helper.ts",
+    ],
+    visibility = ["//docs:__subpackages__"],
+)
+
+ts_library(
```

Comment:
The specs here are not configured.

Reply:
turns out the spec file that was here wasn't used or a spec file.